config/runtime.exs

```diff
@@ -102,6 +102,7 @@ if config_env() == :prod do
 
   config :trento, :alerting,
     enabled: System.get_env("ENABLE_ALERTING", "false") == "true",
+    sender: System.get_env("ALERT_SENDER") || "alerts@trento-project.io",
```

Comment:
````elixir System.get_env("ALERT_SENDER", "alerts@trento-project.io")```

Reply:
right :smile: 
Changed also the recipient to 
```elixir
System.get_env("ALERT_RECIPIENT", "admin@trento-project.io")
```